quiz/quiz.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"bufio"
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"time"
+	"unicode"
+)
+
+type problems struct {
+	q string
+	a string
+}
+
+func main() {
+	const defaultName = "problems.csv"
+	const defaultTimeLimit = 30
+	fileName := flag.String("csv", defaultName, "csv file in the format: `question,answer`")
+	timeLimit := flag.Int("limit", defaultTimeLimit, "Integer time limit ")
+	flag.Parse()
+
+	if isValidFileName(*fileName) {
```

Comment:
Of you can do if not isValidFileName and log the error then do the rest of the code block. This is usually the preferred style

Reply:
I get that this is the preferred style but I'm wondering why that is. Does it improve the code in any way? Is there like a PEP 8 like document I can reference as a style guide?

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow

Reply:
Thanks